client/analysisui/formatdef.js

```diff
@@ -225,6 +256,20 @@ FormatDef.term = new Format ({
         return FormatDef.term._itemToString(raw, 0);
     },
 
+    toAriaLabel: function(raw) {
+        if (raw.length === 1)
+            return raw[0];
+
+        let first = raw[0];
+        if (raw.length > 2) {
+            for (let i = 1; i < raw.length - 1; i++) {
+                first += ', ' + raw[i];
+            }
+        }
+        let second = raw[raw.length - 1];
+        return s_('The interaction of {0} and {1}', [first, second]);
```

Comment:
i think 'human readble list' construction is one of those things that's tricky from an i18n perspective. i don't know if commas are a thing in other languages for example. should we push list construction into an i18n library? and make this `The interaction of {0}`

https://stackoverflow.com/questions/45802189/l10n-i18n-how-to-handle-phrases-with-dynamic-list-of-items